libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/BaseInvokeDialog.cs

```diff
@@ -4,8 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
-using Microsoft.Bot.Builder.Dialogs.Declarative;
-using Microsoft.Bot.Expressions;
+using Microsoft.Bot.Expressions.Properties;
```

Comment:
>Microsoft.Bot.Expressions.Properties; [](start = 6, length = 37)

why not just put this in expressions namespace ?

Reply:
Because I thought that:
1) It would be good to know where they were being used.
2) You don't need them for expressions.